pages/api/user/collect.js

```diff
@@ -6,7 +6,7 @@ export default async (req, res) => {
   const session = await getSession({ req });
 
   // If session exists (thus, user is authenticated)
-  if (session && session.role === 'ADMIN') {
+  if (session && (session.role === 'ADMIN' || session.role === 'JUDGE')) {
```

Comment:
Same as above

Reply:
Why do the judges need to be able to call api/user/collect?  They never need to access the settings page.

Reply:
This is to populate the 'judges' field of each event card iirc in [index.js](https://github.com/uwblueprint/dancefest-web/blob/staging/pages/performances/index.js#L114)